app/file-upload/server/methods/sendFileMessage.js

```diff
@@ -43,15 +44,31 @@ Meteor.methods({
 			title_link_download: true,
 		};
 
+		let thumbId;
+
 		if (/^image\/.+/.test(file.type)) {
 			attachment.image_url = fileUrl;
 			attachment.image_type = file.type;
 			attachment.image_size = file.size;
 			if (file.identify && file.identify.size) {
 				attachment.image_dimensions = file.identify.size;
 			}
+
 			try {
 				attachment.image_preview = await FileUpload.resizeImagePreview(file);
+
+				const thumbBuffer = await FileUpload.createImageThumbnail(file);
+				if (thumbBuffer) {
+					const thumbnail = FileUpload.uploadImageThumbnail(file, thumbBuffer, roomId, user._id);
+					const thumbUrl = FileUpload.getPath(`${ thumbnail._id }/${ encodeURI(file.name) }`);
+					attachment.image_url = thumbUrl;
+					attachment.image_type = thumbnail.type;
+					attachment.image_dimensions = {
+						width: settings.get('Message_Attachments_Thumbnails_Width'),
```

Comment:
oh, we should actually get the new sizes after `sharp` resizes it, the result image will be exactly as the settings 😬

Reply:
bump 🙈

Reply:
OH, didn't modify this one. On it.

Reply:
@sampaiodiego , changed